src/SFA.DAS.Rofjaa.Application/Common/DateTime/DateTimeProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Globalization;
+using SFA.DAS.Rofjaa.Application.Common.DateTime;
+
+namespace SFA.DAS.Rofjaa.Application.Common.DateTime
+{
+    public sealed class DateTimeProvider : IDateTimeProvider
+    {
+        private static readonly IFormatProvider culture = new CultureInfo("en-GB", true);
+
+        public System.DateTime GetNowUtc()
+        {
+            return System.DateTime.UtcNow;
+        }
+
+        public System.DateTime ConvertUtcToUk(System.DateTime utcDateTime)

```

Comment:
What is this for? I couldn't see any usages for this.

Reply:
Removed